pkg/machines/machines.scss

```diff
@@ -155,11 +154,6 @@
     margin: 0.25rem 0 0.25rem 0.5rem;
 }
 
-.machines-network-list {
-    max-height: 500px;
-    padding-bottom: 15px;
-}
```

Comment:
This was a remnant of the previous expand-o-list version of the VM page and it was getting in the way of the details page.

I wonder how much legacy CSS is still in the machines.scss? It's probably worth doing a cleanup as a follow-up PR.